cmd/chihaya/main.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/chihaya/chihaya/server"
 	"github.com/chihaya/chihaya/tracker"
 
+	_ "github.com/chihaya/chihaya/middlewares/ip"
```

Comment:
I had thoughts about putting store-dependent middleware in a subpackage of the store i.e. `github.com/chihaya/chihaya/server/store/middleware/ip` the dependency on the store might appear more obvious then.

Thoughts?